src/Sylius/Bundle/RbacBundle/DependencyInjection/Compiler/TagVoterPass.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\RbacBundle\DependencyInjection\Compiler;
+
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+
+/**
+ * @author Christian Daguerre <christian@daguer.re>
+ */
+class TagVoterPass implements CompilerPassInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function process(ContainerBuilder $container)
+    {
+        if ($container->hasDefinition('sylius.rbac.voter')) {
```

Comment:
I'm not sure, but if you use Symfony decorators, we can delete this compiler pass and tag the base voter instead.